lib/ui/profile/Profile.dart

```diff
@@ -604,18 +609,42 @@ class Profile extends StatelessWidget {
         type: ActionType.simple,
         onTap: () => _pickImage(ImagePickerLib.ImageSource.gallery, viewModel),
       ),
+      ActionSheetListItemViewModel(
+        title: _LocalisedStrings.renameProfile(),
+        type: ActionType.simple,
+        onTap: () => _renameProfileAction(viewModel, context),
+      ),
     ];
 
     final actionSheetViewModel = ActionSheetViewModel(
-      actions, _LocalisedStrings.cancel(),
+      actions,
+      _LocalisedStrings.cancel(),
     );
     return ActionSheet(
       viewModel: actionSheetViewModel,
       style: ActionSheetStyle.defaultStyle(),
     );
   }
 
-  _pickImage(ImagePickerLib.ImageSource imageSource, ProfileViewModel viewModel) {
+  void _renameProfileAction(ProfileViewModel viewModel, BuildContext context) {
+    InputAlert.present(_renameInputAlert(viewModel), context);
+  }
+
+  InputAlert _renameInputAlert(ProfileViewModel viewModel) {
+    return InputAlert(
+      viewModel: InputAlertViewModel(
+          cancelActionText: _LocalisedStrings.cancel(),
+          confirmActionText: _LocalisedStrings.confirm(),
+          titleText: _LocalisedStrings.renameProfile(),
+          hint: _LocalisedStrings.username(),
+          confirmInputAction: (value) {
+            //TODO Add the viewmodel rename func here
```

Comment:
let´s add the interface here and have the TODO in the profile repo? then we don't have to come back the this widget.. so add a rename action too view model, and in the block call a new rename function on the profile repo, and then just have a todo in there.

Reply:
DONE